src/lazyfree.c

```diff
@@ -6,66 +6,55 @@
 static redisAtomic size_t lazyfree_objects = 0;
 static redisAtomic size_t lazyfreed_objects = 0;
 
+/* lazyfree generic function is to call free function, then decrease
+ * lazyfree_objects and increase lazyfreed_objects by specific size. */
+#define lazyfreeGeneric(freeFunction, arg, size) do {  \
+    size_t original_size = size;                       \
```

Comment:
nitpick, I think we should add `()` for them.
```suggestion
    size_t original_size = (size);                     \
```

Reply:
don't think it is necessary, since it is the assignment operation, there is no bad case, i think.